app/src/main/java/com/x8bit/bitwarden/ui/platform/components/button/BitwardenOutlinedButton.kt

```diff
@@ -26,6 +28,8 @@ fun BitwardenOutlinedButton(
     onClick: () -> Unit,
     modifier: Modifier = Modifier,
     isEnabled: Boolean = true,
+    colors: ButtonColors = bitwardenOutlinedButtonColors(),
+    outlineBorderColor: Color = BitwardenTheme.colorScheme.outlineButton.border,
```

Comment:
would you suggest then making a different button composable or like a `BitwardenButtonColors` object which as the outlineColor as part of it?

Reply:
It would litterally just wrap the `outlineBorderColor` and the entire `ButtonColors`.

Looking for opinions

Reply:
I would say a new object is necessary to do something similar, since the outline color does not exist in the `ButtonColors` object from M3. Making this change so we can see what it looks like.